packages/@guardian/source-react-components-development-kitchen/components/alert-switch/AlertSwitch.tsx

```diff
@@ -0,0 +1,94 @@
+import { css } from '@emotion/react';
+import { ArticlePillar, ArticleTheme } from '@guardian/libs';
+import { textSans } from '@guardian/src-foundations/typography';
+import type { Props } from '@guardian/src-helpers';
+import { decideBackground, alertSwitchStyles } from './styles';
+
+// https://docs.google.com/spreadsheets/d/1QUa5Kh734J4saFc8ERjCYHZu10_-Hj7llNa2rr8urNg/edit?usp=sharing
```

Comment:
I get an error saying this file has been deleted. Not sure if it has or if it's a permissions issue?

Reply:
Thanks Jamie, I had at first copied one of the existing components and then modified it to create the alert switch. I had missed deleting this part, just deleted it :)